packages/docs/pages/Form/FormPage.tsx

```diff
@@ -31,7 +31,7 @@ export default () => (
     <Text>
       Form fields are essential to any website or web application. <Code>id</Code>'s' will be auto-generated for
       input/textarea and labels <Code>for</Code> attribute, unless manually specifying one.{' '}
-      <Link href="https://bigcommerce.design/components/fields" target="_blank">
+      <Link href="https://design.bigcommerce.com/components/form-fields" target="_blank">
```

Comment:
@chanceaclark this is just forms now. https://design.bigcommerce.com/components/forms